src/utils/update/review_trusts.ts

```diff
@@ -120,3 +120,19 @@ export const moveTrustOutOfReview = (appData: ApplicationData) => {
 
   appData.trusts?.push(trust);
 };
+
+export const putTrustInChangeScenario = (appData: ApplicationData, trustId: string, trusteeType?: string) => {
+  const trustForChangeScenario = appData.trusts?.splice(appData.trusts.findIndex(trust => trust.trust_id === trustId), 1)[0] as Trust;
```

Comment:
some of the values come from the user. So the trust_id could be not found. Or trustee type not something expected.

Reply:
The user will never be able to see or edit the URL as it is just an orchestrator call.

Reply:
We have middleware too on these routes, right? `hasTrustWithId` and `hasTrusteeWithId` or something similar, that'll verify the parameters given are valid before the request reaches the controller.

Reply:
Exactly @conuirchgov